src/sentry/api/serializers/models/group.py

```diff
@@ -973,6 +978,57 @@ def get_attrs(self, item_list, user):
                     attrs[item].update({"filtered_stats": filtered_stats[item.id]})
                 attrs[item].update({"stats": stats[item.id]})
 
+            if self._expand("sessions"):
+                cache_keys = list(
+                    {self._build_session_cache_key(item.project_id) for item in item_list}
+                )
+                cache_data = cache.get_many(cache_keys)
+                missed_items = []
+                for item, cache_key in zip(item_list, cache_keys):
```

Comment:
You can't really do `zip(item_list, cache_keys)` can you? `item_list` is a list of groups, but `cache_keys` will be the unique cache keys for projects in those groups, and so will have a different number of rows.

I think what you need to do is take the list of unique project_ids, then zip `project_ids, cache_keys` and convert them to a dict. Then iterate over  `item_list` and add any groups without a project to the `missed_items` list etc.

Reply:
Oops you're right. I fixed this.